src/main/resources/org/nest/sympy/misc.py

```diff
@@ -0,0 +1,178 @@
+from sympy.parsing.sympy_parser import parse_expr
+from sympy import *
+
+from sympy.matrices import zeros
+
+# Define constants
+MAX_ORDER = 10
```

Comment:
As far as I understood, it is anyway very unlikely to need 10 here. An explanatory comment should be added, though. The same applies to the line below.